updater/internal/path/path_linux.go

```diff
@@ -14,12 +14,53 @@
 
 package path
 
-import "go.uber.org/zap"
+import (
+	"os/exec"
+	"path/filepath"
+
+	"go.uber.org/zap"
+)
 
 // LinuxInstallDir is the install directory of the collector on linux.
 const LinuxInstallDir = "/opt/observiq-otel-collector"
 
+// SystemdFilePath is the path for systemd service
+const SystemdFilePath = "/usr/lib/systemd/system/observiq-otel-collector.service"
+
+// SysVFilePath is the path for sysv service
+const SysVFilePath = "/etc/init.d/observiq-otel-collector"
+
 // InstallDir returns the filepath to the install directory
 func InstallDir(_ *zap.Logger) (string, error) {
 	return LinuxInstallDir, nil
 }
+
+// LinuxServiceCmdName returns the filename of the service command available
+// on this Linux OS. Will be one of systemctl and service
+func LinuxServiceCmdName() string {
+	var path string
+	var err error
+	path, err = exec.LookPath("systemctl")
+	if err != nil {
+		path, err = exec.LookPath("service")
+	}
+	if err != nil {
+		// Defaulting to systemctl in the most common path
+		// This replicates prior behavior where this was
+		// a static define
+		path = "/usr/bin/systemctl"
+	}
+	_, filename := filepath.Split(path)
+	return filename
+}
+
+// LinuxServiceFilePath returns the full path to the service file
+func LinuxServiceFilePath() string {
+	if LinuxServiceCmdName() == "systemctl" {
+		return SystemdFilePath
+	} else if LinuxServiceCmdName() == "systemctl" {
+		return SysVFilePath
+	}
+
+	return SystemdFilePath
+}
```

Comment:
As this is written, looks like it will always redurn systemdFilePath.
```suggestion
func LinuxServiceFilePath() string {
        serviceCmd := LinuxServiceCmdName()
	
	if serviceCmd == "service" {
		return SysVFilePath
	}

	return SystemdFilePath
}
```